### DISCUSSION TITLE
Is there a way to filter for two WordPress websites based on the Branded Career Portal Posting feature?

### DISCUSSION THREAD

**Discussion by holty07** [bullhorn/career-portal#438] [Q&A] [answered]
Hi all,

As title says, I have a client that has two websites that they are posting jobs to using the OSCP. The differentiation they have made between them is using two different Branded Career Portal Posting accounts as per the articles [here ](https://help.bullhorn.com/bhatsTopics/s/article/ATS-Enhancing-and-Customizing-the-OSCP)and [here](https://help.bullhorn.com/bhatsTopics/s/article/ATS-How-do-I-add-a-new-brand-for-Indeed-posting). We are not able to find a way to filter on the websites based on the selection here. Is this even possible or do we need to get custom fields created?

**Comment by pbearne** [comment]
I am not sure if you can do it with OSCP but if find you can't [Matador Jobs](https://matadorjobs.com/) can filter on any value in the Job object

I hope this helps
Paul

**Reply by holty07** [reply]
Hmm, thanks. I will have a look at that.

**Comment by charlesabarnes** [comment, marked as answer]
Hello, 

I do not believe that the field is persisted in a way that is accessible via the REST API.  I suggest utilizing a field such as a customText field or something similar to differentiate between the "brands" on the publishing side.   This will be required for any portal that utilizes the REST API

**Reply by holty07** [reply]
A custom field seems to be the only way to do this.
